fdbclient/include/fdbclient/TenantEntryCache.actor.h

```diff
@@ -151,12 +296,23 @@ class TenantEntryCache : public ReferenceCounted<TenantEntryCache<T>>, NonCopyab
 			return ret;
 		}
 
-		TraceEvent(SevInfo, "TenantEntryCacheGetByIdRefresh").detail("TenantId", tenantId);
+		if (!tenantsEnabled(cache)) {
+			// If tenants are disabled on the cluster avoid using the cache
+			cache->misses += 1;
```

Comment:
nit: do we want to increment `missCounter` on a cache miss?

Reply:
Sure i can remove